common/src/main/java/xyz/wagyourtail/jsmacros/client/api/library/impl/FWorld.java

```diff
@@ -88,6 +93,42 @@ public boolean isWorldLoaded() {
         return mc.world != null;
     }
 
+    /**
+     * Example usage: <br>
+     * {@code World.getWorldScanner(World.getBlockAnyFilter("CONTAINS").addOption("diamond_ore", "redstone_ore"), null)}.
+     *
+     * @return a scanner for the current world
+     *
+     * @since 1.6.4
+     */
+    public WorldScanner getWorldScanner(Function<Block, Boolean> blockFilter, Function<BlockState, Boolean> stateFilter) {
```

Comment:
never expose Functional Interfaces directly to the user or they will accidentally try using js functions (which will work until they cause a concurrency exception)